packages/blinks-core/src/service-api/types.ts

```diff
@@ -0,0 +1,19 @@
+export interface Provider {
```

Comment:
BlinkProvider? or just Provider?

Reply:
Technically BlinkProvider, cc @tsmbl.

Either way, we should keep the types consistent between the API docs and SDK, so including Alexey here as well

Reply:
@tsmbl and I agreed to rename to BlinkProvider. Done